autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/BasicSelect/BasicOptionRow.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { Typography, Select, MenuItem } from '@material-ui/core';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from '../../../../../../redux/reducer';
+import { updateCourseCard } from '../../../../../../redux/actions/courseCards';
+import * as styles from './BasicSelect.css';
+
+interface BasicOptionRowProps {
+    id: number;
+    value: 'honors' | 'web';
+}
+
+/**
+ * Renders one row in the BasicSelect table
+ * @param props include id of the course card and value, which should be the name of the
+ * option selected by this row, formatted as it is found in the Redux course cards
+ */
+const BasicOptionRow: React.FC<BasicOptionRowProps> = ({ id, value }) => {
+  const option = useSelector<RootState, string>((state) => state.courseCards[id][value] || 'exclude');
+  const dispatch = useDispatch();
+
+  return (
+    <tr>
+      <td>
+        <Typography variant="body1" style={{ paddingRight: 8 }} id={`honors-${id}`}>
```

Comment:
Change this and the `labelId` on the Select to use `value` instead of always say honors